infrastructure/local/peer-docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
       AUTH_DATABASE_URL: postgresql://peerauth:peerauth@database/peerauth
       INTROSPECTION_HTTPSIG: "false"
       AUTH_SERVER_DOMAIN: "http://localhost:4006"
+    depends_on:
+      - auth
```

Comment:
Sometimes, when starting up the `localenv`, I would get:
```bash
max ~/interledger/rafiki [1092/mk/local-env-psql] $ pnpm localenv up -d --build

> @interledger/rafiki@1.0.0 localenv /Users/max/interledger/rafiki
> docker compose -f ./infrastructure/local/docker-compose.yml -f ./infrastructure/local/peer-docker-compose.yml "up" "-d" "--build"

network local_rafiki declared as external, but could not be found

 ELIFECYCLE  Command failed with exit code 1.
```

Retrying the command would work, meaning this fails because sometimes, the peer docker-compose would try to find the external network that hasn't been created yet in the main docker-compose file

Reply:
- https://github.com/interledger/rafiki/issues/1003